src/reducers/entries_reducers.js

```diff
@@ -16,6 +16,12 @@ const entriesReducer = (state = {}, action) => {
       const { entry } = action.payload;
       newState[entry.id] = entry;
       return newState;
+    case RECEIVE_CONTESTS:
+      let winningEntries = Object.keys(action.payload.contests).map( (contest) => {
+        return action.payload.contests[contest].winning_entry;
+      });
+      newState = merge({}, state, winningEntries);
```

Comment:
does this work? we would want to these entries to go in under keys of their ids to be consistent, this looks to me like you are merging in an array to the state rather than an object?

Reply:
Ha! Weirdly, it sort of works. But not very well; I just blanked that it's an array. Will refactor momentarily.

Reply:
Please review again! Let me know if you see more issues.